rotkehlchen/tasks/calendar.py

```diff
@@ -138,10 +138,25 @@ def maybe_create_ens_reminders(database: DBHandler) -> None:
                 color=None,
                 auto_delete=True,
             ))
-        elif db_calendar.count_reminder_entries(  # else calendar entry already exists
-            event_id=(calendar_entry_id := calendar_entries['entries'][0].identifier),
-        ) > 0:  # and already has a reminder entry
-            continue  # we don't add any automatic reminders
+        else:  # else calendar entry already exists
+            calendar_entry = calendar_entries['entries'][0]
+            if ens_expires > calendar_entry.timestamp:  # if a later expiry is found
+                db_calendar.update_calendar_entry(CalendarEntry(
+                    identifier=calendar_entry.identifier,
+                    name=f'{ens_name} expiry',
+                    timestamp=ens_expires,  # update the calendar entry
+                    description=f'{ens_name} expires on {customizable_date.timestamp_to_date(ens_expires)}',  # noqa: E501
```

Comment:
can you assign the description to a variable? The reason is that when you modify it you don't need to change multiple places